docker-compose.dind.yml

```
@@ -0,0 +1,25 @@
+
+  playwright:
+    container_name: website-playwright
+
+  apollo:
+    container_name: website-apollo
+
+  mockoon:
+    container_name: website-mockoon
+
+  k6:
+    container_name: website-k6 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)